website/docs/usage/embeddings-transformers.md

```diff
@@ -714,10 +714,13 @@ subnetwork** within one of your components, and add additional layers to build a
 network for a temporary task that forces the model to learn something about
 sentence structure and word cooccurrence statistics.
 
-Pretraining produces a **binary weights file** that can be loaded back in at the
-start of training, using the configuration option `initialize.init_tok2vec`.
-The weights file specifies an initial set of weights. Training then proceeds as
-normal.
+Pretraining produces a **binary weights file** that can be loaded back in at
+the start of training, using the configuration option
+`initialize.init_tok2vec`.  The weights file specifies an initial set of
+weights. Training then proceeds as normal. Note that if you fill this value in
+before running pretraining, the command will fail because the file doesn't
+exist yet. You can just update your config file after pretraining finishes, or
```

Comment:
This is kind of unfortunate because we tell people to use the exact same config for training and pretraining :( Not sure how better to deal with this though. At which point does it crash when providing an unexisting pretraining path during training, and can we prevent that from happening?

Reply:
It gives a ConfigValidationError from [this line](https://github.com/explosion/spaCy/blob/24cdd4c88e291d01aff9cd92c37c6cdb03479729/spacy/training/initialize.py#L186). I don't think that line currently has access to whether it's being called in pretraining or not, so the exception might need to be handled higher up.

Also, I think we technically support pretraining with seed tok2vecs. So you can start pretraining with the output of a different pretraining... If we support that, it's hard to know if a non-existing file is a mistake or a file that hasn't been created yet, though I guess we could just print a warning.

Reply:
I think we can remove this edit now, no?

Reply:
Good point! Let me remove that...